master/docs/tutorial/firstrun.rst

```diff
@@ -51,27 +51,36 @@ Creating a master
 -----------------
 
 The first necessary step is to create a virtualenv for our master.
-All our operations will happen in this directory:
+We will also use a separate directory to demonstrate the separation between a
+master and worker::
 
 .. code-block:: bash
 
   cd
   mkdir tmp
   cd tmp
-  virtualenv --no-site-packages bb-master
-  cd bb-master
+  mkdir buildbot
+  cd buildbot
+  virtualenv --no-site-packages sandbox
+  source sandbox/bin/activate
+
+
+.. note:: *Python 3*
+
+    Python 3 support is work in progress, if you default interpreter is Python
+    3, then you must pass pass the ``--python=python3`` to ``virtualenv``.
 
 Now that we are ready, we need to install buildbot:
 
 .. code-block:: bash
 
-  ./bin/pip install buildbot[bundle]
+  pip install buildbot[bundle]
```

Comment:
while you are at it, please add a note here to update pip. We often have user having issues because of older pip